connectivity/lwipstack/include/lwipstack/lwipopts.h

```diff
@@ -311,6 +311,9 @@
 
 #endif
 
+// FIXME: Add compile time configuration and define conditionaly
+#define LWIP_NETBUF_RECVINFO            1
```

Comment:
Can you add a compile time configuration parameter for this ?

Reply:
Done